samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/facade/PublishedAssessmentFacadeQueries.java

```diff
@@ -1651,6 +1651,29 @@ public HashMap getTotalSubmissionPerAssessment(String agentId, String siteId) {
 		}
 		return h;
 	}
+	
+    /**
+     * return submission numbers for the assessment by giving the publishedAssessmentId
+     * Added by -Qu for assessment deletion safe check SAM-2587
+     */
+	public Integer getTotalSubmissionForEachAssessment(
+			final Long publishedAssessmentId) {
```

Comment:
Can you move the argument up a line into the method signature? It's only one argument, doesn't really need to be on a newline.